### PR TITLE
Fix regex by adding slashes for fluentd grep filter

### PR DESCRIPTION
**Pull request by PettitWesley**
By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: fluentd-template.go [hunk 1]
```diff
@@ -33,7 +33,7 @@ var fluentDConfigTemplate = `{{- range .IncludeConfigHeadOfFile -}}
     @type  grep
     <regexp>
         key {{ .Key }}
-        pattern {{ .Regex }}
+        pattern /{{.Regex}}/
```
Comment (fenxiong) [inline fluentd-template.go]: hmm i think we should let customer specify the slash.. from https://docs.fluentd.org/filter/grep, "The pattern parameter is string type before 1.2.0." so i don't think we can use slash for version before 1.2

Comment (PettitWesley) [reply fluentd-template.go]: Hmm I see, I'll test this filter with >1.2 and <1.2 to verify this is what happens.

Comment (PettitWesley) [reply fluentd-template.go]: I tested with and without slashes with v1.0 and v1.5.

v1.5 had the same behavior with and without the slashes, v1.0 only worked without the slashes as expected.

**Comment by PettitWesley** [comment]
Closing due to: https://github.com/awslabs/go-config-generator-for-fluentd-and-fluentbit/pull/2#discussion_r298817769

**Pull request closed without merging**